testing/README.md

```diff
@@ -0,0 +1,44 @@
+
+# Testing and Interoperability
+
+## rnaget API Compliance Testing
+
+Implementors can check if their rnaget implementations conform to the specification by using our [compliance suite](https://github.com/ga4gh-rnaseq/rnaget-compliance-suite).
+
+Several sources have written test servers and clients for the rnaget API.  These are available for testing.  Testing configuration files required for the compliance tests can be found in this repository.  A combined configuration file for testing all available servers is also provided.
+
+Implementors who wish to be included in this list can do so by providing their configuration file.
+
+### Compliance report
```

Comment:
The rnaget-compliance application will actually start the test server automatically if the `--serve` option is specified. Should we mention that? The commands below only need to be run if the server is stopped, and one wants to restart the server with the same archive of web files.